x/group/testutil/expected_keepers.go

```diff
@@ -8,13 +8,13 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/group"
 )
 
-// AccountKeeper extends `AccountKeeper` from expected_keepers.
+// extends `AccountKeeper` from expected_keepers.
```

Comment:
I think it is the guideline to start with the function/interface name in go docs.